khtools/common.py

```diff
@@ -23,7 +22,8 @@ def get_figure_folder():
 def load_tabula_muris_annotations():
     """Fetch latest FACS annotations of Tabula Muris data from GitHub"""
     annotations = pd.read_csv(
-        'https://github.com/czbiohub/tabula-muris/raw/master/00_data_ingest/18_global_annotation_csv/annotations_facs.csv',
+        'https://github.com/czbiohub/tabula-muris/raw/master/00_data_ingest/'
```

Comment:
this would be a good thing to also add to a config file, especially since it might change in the future